functions/Get-DbaLinkedServer.ps1

```diff
@@ -0,0 +1,90 @@
+﻿function Get-DbaLinkedServer
+{
+<#
+.SYNOPSIS
+Gets all linked servers and summary of information from the sql servers listed
+
+.DESCRIPTION
+Retrieves information about each linked server on the instance
+
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER Databases
+Return information for only specific databases
+
+.NOTES
+Author: Stephen Bennett ( https://sqlnotesfromtheunderground.wordpress.com/ )
+	
+dbatools PowerShell module (https://dbatools.io)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	
+.LINK
+https://dbatools.io/Get-DbaLinkedServer
+
+.EXAMPLE
+Get-DbaLinkedServer -SqlServer DEV01
+
+Returns all Linked Servers for the SQL instance DEV01
+
+#>
+	[CmdletBinding()]
+	param (
+		[Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[string[]]$SqlInstance,
+		[System.Management.Automation.PSCredential]$SqlCredential
+	)
+    process
+    {
+        foreach ($Instance in $SqlInstance)
+        {
+            try
+	        {
+	            Write-Verbose "Connecting to $Instance"
+                $server = Connect-SqlServer -SqlServer $Instance -SqlCredential $sqlcredential
+	        }
+	        catch
+	        {
+	            Write-Warning "Failed to connect to: $srv"
+                break
+	        }
+            if($server -contains "\")
+            {
+                $computername = ($server.Name).Split("\")[0]
+                $inst =  ($server.Name).Split("\")[1]
+            }
+            else
+            {
+                $computername = $server.Name
+                $inst =  ""
```

Comment:
As well you can actually  just remove the else all together. 

Also you don't need to use the extra parenthesizes to split it, and just another way to do it:

```powershell
if ($server.Contains("\") {
   $computername = $server.Name.Split("\")[0]
   $inst = $server.Name.Split("\")[1]
}
```

The above will ensure you still get the computer name, and if you deal with a default instance, the `$inst` will be `null` and not an empty string.